parsers/ocaml.c

```diff
@@ -91,102 +96,108 @@ typedef enum {
-	{ "exception" , OcaKEYWORD_exception }, 
-	{ "for"       , OcaKEYWORD_for       }, 
-	{ "fun"       , OcaKEYWORD_fun       }, 
-	{ "function"  , OcaKEYWORD_fun       }, 
-	{ "functor"   , OcaKEYWORD_functor   }, 
-	{ "in"        , OcaKEYWORD_in        }, 
-	{ "let"       , OcaKEYWORD_let       }, 
-	{ "match"     , OcaKEYWORD_match     }, 
-	{ "method"    , OcaKEYWORD_method    }, 
-	{ "module"    , OcaKEYWORD_module    }, 
-	{ "mutable"   , OcaKEYWORD_mutable   }, 
-	{ "object"    , OcaKEYWORD_object    }, 
-	{ "of"        , OcaKEYWORD_of        }, 
-	{ "rec"       , OcaKEYWORD_rec       }, 
-	{ "sig"       , OcaKEYWORD_sig       }, 
-	{ "struct"    , OcaKEYWORD_struct    }, 
-	{ "then"      , OcaKEYWORD_then      }, 
-	{ "try"       , OcaKEYWORD_try       }, 
-	{ "type"      , OcaKEYWORD_type      }, 
-	{ "val"       , OcaKEYWORD_val       }, 
-	{ "value"     , OcaKEYWORD_value     }, /* just to handle revised syntax */
-	{ "virtual"   , OcaKEYWORD_virtual   }, 
-	{ "while"     , OcaKEYWORD_while     }, 
-	{ "with"      , OcaKEYWORD_with      }, 
-
-	{ "or"        , Tok_Op               }, 
-	{ "mod "      , Tok_Op               }, 
-	{ "land "     , Tok_Op               }, 
-	{ "lor "      , Tok_Op               }, 
-	{ "lxor "     , Tok_Op               }, 
-	{ "lsl "      , Tok_Op               }, 
-	{ "lsr "      , Tok_Op               }, 
-	{ "asr"       , Tok_Op               }, 
-	{ "->"        , Tok_To               }, 
-	{ "true"      , Tok_Val              }, 
-	{ "false"     , Tok_Val              }
+	{ "and", OcaKEYWORD_and },
+	{ "begin", OcaKEYWORD_begin },
+	{ "class", OcaKEYWORD_class },
+	{ "do", OcaKEYWORD_do },
+	{ "done", OcaKEYWORD_done },
+	{ "else", OcaKEYWORD_else },
+	{ "end", OcaKEYWORD_end },
+	{ "exception", OcaKEYWORD_exception },
+	{ "for", OcaKEYWORD_for },
+	{ "fun", OcaKEYWORD_fun },
+	{ "function", OcaKEYWORD_function },
+	{ "functor", OcaKEYWORD_functor },
+	{ "if", OcaKEYWORD_if },
+	{ "in", OcaKEYWORD_in },
+	{ "let", OcaKEYWORD_let },
+	{ "match", OcaKEYWORD_match },
+	{ "method", OcaKEYWORD_method },
+	{ "module", OcaKEYWORD_module },
+	{ "mutable", OcaKEYWORD_mutable },
+	{ "object", OcaKEYWORD_object },
+	{ "of", OcaKEYWORD_of },
+	{ "rec", OcaKEYWORD_rec },
+	{ "sig", OcaKEYWORD_sig },
+	{ "struct", OcaKEYWORD_struct },
+	{ "then", OcaKEYWORD_then },
+	{ "try", OcaKEYWORD_try },
+	{ "type", OcaKEYWORD_type },
+	{ "val", OcaKEYWORD_val },
+	{ "value", OcaKEYWORD_value }, /* just to handle revised syntax */
+	{ "virtual", OcaKEYWORD_virtual },
+	{ "while", OcaKEYWORD_while },
+	{ "with", OcaKEYWORD_with },
+
+	{ "or", Tok_Op },
+	{ "mod ", Tok_Op },
+	{ "land ", Tok_Op },
+	{ "lor ", Tok_Op },
+	{ "lxor ", Tok_Op },
+	{ "lsl ", Tok_Op },
+	{ "lsr ", Tok_Op },
+	{ "asr", Tok_Op },
+	{ "->", Tok_To },
+	{ ":", Tok_Of },
+	{ "true", Tok_Val },
+	{ "false", Tok_Val }
 };
 
 static langType Lang_Ocaml;
 
-static boolean exportLocalInfo = FALSE;
+boolean exportLocalInfo = FALSE;
```

Comment:
Good question. I have no good answer, and changing it back doesn't break anything, so I've reverted this change.